project-base/storefront/package.json

```diff
@@ -71,10 +74,11 @@
         "@graphql-codegen/near-operation-file-preset": "^3.0.0",
         "@graphql-codegen/typescript-operations": "~4.0.1",
         "@graphql-codegen/typescript-urql": "~4.0.0",
-        "@mswjs/interceptors": "0.19.0",
+        "@mswjs/interceptors": "0.20.0",
```

Comment:
Was it necessary to upgrade this package? Not saying that it is wrong, but how does it relate to these changes?

Reply:
It fixed (partially) the local errors I was having. I tracked the issue to this version of the package, so I tried to upgrade it and the response error stopped.

Reply:
But since only my local is broken, I will downgrade it to its original version number so we don't introduce more bugs along the way.